corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/request.less

```diff
@@ -6,3 +6,55 @@
 .formplayer-request {
     cursor: pointer;
 }
+
+.pagination-form {
+    height: 40px;
+}
+
+.pagination-user-form {
```

Comment:
I'm a little surprised that the existing bootstrap and web apps styles weren't sufficient for this UI. For consistency's sake, we only add new styles when there's an effect we can't achieve with bootstrap's classes or existing HQ classes. Can you take a pass at only styling with boostrap/HQ - you might take a look at other web apps styles, like `module-search-container` might be helpful for keeping the pagination content within the bounds of the white background? Or if you already tried using standard classes and ran into problems, let's talk about them.